### PR TITLE
Add react-native-page-control, react-native-paper, react-native-fbads, react-native-material-palette, react-native-universal-picker

### PR DESCRIPTION
**Pull request by ubatin**
Hi, libraries for iOS and Android.
- [react-native-page-control](https://github.com/silentcloud/react-native-page-control)
- [react-native-paper](https://github.com/callstack/react-native-paper)
- [react-native-fbads](https://github.com/callstack/react-native-fbads) 
- [react-native-material-palette](https://github.com/callstack/react-native-material-palette) 
- ~~[react-native-opentok](https://github.com/callstack/react-native-opentok)~~
- [react-native-universal-picker](https://github.com/jkomyno/react-native-universal-pickerr)

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -255,6 +262,13 @@
     "expo": true
   },
   {
+    "githubUrl": "https://github.com/jkomyno/react-native-universal-pickerr",
```
Comment (jimmylee) [inline react-native-libraries.json]: Theres an extra `r` here.

File: react-native-libraries.json [hunk 2]
```diff
@@ -1733,5 +1747,33 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-material-palette",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-opentok",
```
Comment (jimmylee) [inline react-native-libraries.json]: This library is a little dated, have you used it? If you think it's good I'll take your word for it.

Comment (ubatin) [reply react-native-libraries.json]: Actually I'm not using this library. So I don't known that this library is good or bad. If you don't need this library I'll remove it. :+1:

File: react-native-libraries.json (hunk 1 above)
Comment (ubatin) [reply react-native-libraries.json]: Removed extra `r` https://github.com/react-community/native-directory/pull/60/commits/0b806fa6426ca747179a891fb94a40fd537abc36 .

File: react-native-libraries.json (hunk 2 above)
Comment (jimmylee) [reply react-native-libraries.json]: @ubatin so for the future we should only add libraries we personally like or use. That helps us keep the qualities of libraries listed above average.

Comment (jimmylee) [reply react-native-libraries.json]: lets remove this one.

Comment (ubatin) [reply react-native-libraries.json]: @jimmylee , thank your review. I'll remove this library.

Comment (ubatin) [reply react-native-libraries.json]: hi @jimmylee , I removed this library :tada:

**Comment by jimmylee** [comment]
Thanks :D

**Pull request merged**